index.html

```diff
@@ -53,5 +54,237 @@ <h1 >
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <div class="btn">
+                        <button class="greenbtn">See project</button>
+                    </div>
+                </div>
+            </div>
+      
+
+        <div class="project">
+            <div class="image"></div>
+            <div class="content">
+                <h3>
+                    Multi-Post Stories <br>
+                    Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>HTML</li>
+                </ul>
+                <div class="btn">
+                    <button class="greenbtn">See project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="project">
+            <div class="image"></div>
+            <div class="content">
+                <h3>
+                    Multi-Post Stories <br>
+                    Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>HTML</li>
+                </ul>
+                
+                <div class="btn">
+                    <button class="greenbtn">See project</button>
+                </div>
+
+
+
+            
+            
+             
+            </div>
+
+        </div>
+        
+        <div class="project">
+            <div class="image"></div>
+            <div class="content">
+                <h3>
+                    Multi-Post Stories <br>
+                    Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>HTML</li>
+                </ul>
+                
+                <div class="btn">
+                    <button class="greenbtn">See project</button>
+                </div>
+
+
+
+            
+            
+             
+            </div>
+
+        </div>
+
+        <div class="project">
+            <div class="image"></div>
+            <div class="content">
+                <h3>
+                    Multi-Post Stories <br>
+                    Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>HTML</li>
+                </ul>
+                
+                <div class="btn">
+                    <button class="greenbtn">See project</button>
+                </div>
+
+
+
+            
+            
+             
```

Comment:
- [ ] Kindly I recommend you to remove the the empty spaces to make your code base clean.